packages/grep-workspace/src/node/grep-workspace.spec.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as chai from 'chai';
+import * as stream from 'stream';
+
+import { testContainer } from './inversify.spec-config';
+import { GrepProcessFactory } from './grep-workspace';
+
+const expect = chai.expect;
+
+describe("grep-workspace", () => {
+    const grepProcessFactory = testContainer.get<GrepProcessFactory>(GrepProcessFactory);
+
+    it('test pipe stream', function () {
+        const grepWorkSpace = grepProcessFactory({ command: process.execPath, 'args': ['--version'] });
+        const outStream = new stream.PassThrough();
+
+        const p = new Promise<String>((resolve, reject) => {
+            outStream.on('data', data => {
+                console.log("data received is : " + data.toString());
```

Comment:
we should not leave that here

Reply:
i want to keep it for now to see the output of the command. I added a note to remove it in the end.